WMF Framework/Significant Events Endpoint/ArticleAsLivingDocViewModels.swift

```diff
@@ -241,7 +333,22 @@ public extension ArticleAsLivingDocViewModel {
         
         public func hash(into hasher: inout Hasher) {
```

Comment:
I'm a bit afraid to touch this since it's working - so feel free to ignore - but using `subtitleTimestampDisplay` for `SectionHeader`'s equality check and hash still feels odd, like it should be based on a more concrete identifier rather than a generated string meant to be displayed in UI. This was existing and my own doing, but if you want to I would welcome an attempt at this cleanup in your timestamp PR (especially if you find you need to rework `subtitleTimestampDisplay` anyways).

Reply:
I was definitely in "don't break, just make it work mode" on this PR, too. Same note above about giving this a try in the timestamp PR.